File: app/src/main/java/me/tysheng/xishi/utils/LogUtil.java
package me.tysheng.xishi.utils;

import android.util.Log;

import me.tysheng.xishi.BuildConfig;

/**
 * Created by Sty
 * Date: 16/8/27 13:09.
 */
public class LogUtil {
    private static final String TAG = "sty";
    private static boolean Debug = BuildConfig.DEBUG;

    public static void d(String s){
        if (Debug){
            Log.d(TAG,s);
        }
    }
}

File: app/src/main/java/me/tysheng/xishi/utils/LogUtil.kt
package me.tysheng.xishi.utils

import android.util.Log

import me.tysheng.xishi.BuildConfig

/**
 * Created by Sty
 * Date: 16/8/27 13:09.
 */
object LogUtil {
    private val TAG = "sty"
    private val Debug = BuildConfig.DEBUG

    fun d(s: String?) {
        if (Debug) {
            Log.d(TAG, s)
        }
    }
}
